### ISSUE TITLE
Stable release 

### ISSUE THREAD

**Issue by JohnRSim** [sjefferson99/enviro#7]
Does this repo have all of your enviro weather updates?

I was looking at the enviro repo but there are a lot of pull requests still waiting to be merged. And last update was 7months ago .

Thanks
J

**Comment by sjefferson99** [comment]
Hi @JohnRSim 
This forked repo houses all the branches I have used to make the PRs against the Pimoroni repo. It also has some extra branches I am using to develop my own fork of the code:
- sj-enviro: All the branches that are for PRs to the Pimoroni repo and potentially any extra dev I have done and tested, the releases in this repo are against this branch and are the nearest thing to stable given I am only testing on my own board.
- sj-develop: My development branch where individual features have been merged and I am testing on my board ahead of merging into sj-enviro if I don't see any issues
- sj-develop-multicore: A branch including all my experimental features for using multicore to run continuous polls at around 4hz for more accurate recording. This is currently sitting alongside the enviro code, but is likely to completely replace it and will get its own repo once this has settled down a bit.

If you want to try out one or more of the features I have outstanding PRs for, then I recommend using the code from the latest release in this repo i.e. sj-enviro branch code. Feel free to raise an issue if you experience any problems with that code, I make no promises, but it should work and I'll do my best to look into it if it doesn't.
